eng/testing/linker/project.csproj.template

```diff
@@ -8,4 +8,23 @@
     <TargetingPackDir>{TargetingPackDir}</TargetingPackDir>
   </PropertyGroup>
 
+  <!-- Logic to override the default IlLink tasks that come from the SDK and use the one
+  we use in dotnet/runtime repo -->
+
+  <PropertyGroup>
+    <_ILLinkTasksDirectoryRoot>$(PkgMicrosoft_NET_ILLink_Tasks)\tools\</_ILLinkTasksDirectoryRoot>
+    <_ILLinkTasksTFM>netcoreapp3.0</_ILLinkTasksTFM>
```

Comment:
Why netcoreapp3.0 here?

Reply:
ILLink package is packaged as a tool, and tools have TFMs that they target. ILLink specifically multitargets for net472 and netcoreapp3.0, and given we are running the .NET Core msbuild we want to use the netcoreapp3.0 one. This is also the same way the regular linker that comes with the sdk work today.